cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -102,33 +102,13 @@ public void runTool(VersionIdentifier toolVersion, String... args) {
   /**
    * @return the {@link Path} where the main executable file of this tool is installed.
    */
-  public Path getToolBinary() {
+  public abstract Path getToolBinary();
```

Comment:
There are two things to distinguish for global tools:
1. the installer binary used to install the tool globally on the system
2. the binary of the tool once it is installed on the system

According to the JavaDoc this method should refer to the 2nd one.
However, I can not see how we could implement this in a generic way.
Also we have to revisit the sense of this method after #111 - most probably this method will disappear (Sorry, when stories are implemented in parallel that have overlapping code changes it always gets kind of tricky and you will most probably also will have to deal with git conflicts. It therefore will make sense to work closer together with @aBega2000 so you both work together and not "against each other").

Reply:
Im waiting for https://github.com/devonfw/IDEasy/pull/111 to be merged then I'll resolve the conflicts and change my code according to his changes.

Reply:
I merged #111 now.

Reply:
Method is gone, resolved.